docs/tutorial.rst

```diff
@@ -363,16 +362,16 @@ Let's do the bootstrap test on the two categories.
 
 .. ipython:: python
 
-    num_nonsmokers = smoker_and_wt.where('m_smoker', 0).num_rows
+    num_nonsmokers = smoker_and_wt.where('Maternal Smoker', 0).num_rows
     def bootstrap_once():
         """
         Computes one bootstrapped difference in means.
         The table.sample method lets us take random samples.
         We then split according to the number of nonsmokers in the original sample.
         """
         resample = smoker_and_wt.sample(with_replacement = True)
-        bootstrap_diff = resample.column('birthwt')[:num_nonsmokers].mean() - \
-            resample.column('birthwt')[num_nonsmokers:].mean()
+        bootstrap_diff = resample.column('Birth Weight')[:num_nonsmokers].mean() - \
+            resample.column('Birth Weight')[num_nonsmokers:].mean()
```

Comment:
I'm not familiar with how to deploy this locally (due to it being a Rust file), but the changes made seem to be 100% correct to make. Let's deploy and verify!

Reply:
I built the docs with `make docs`, `make serve_docs` and then viewed in my browser at `localhost:8000` and the result looks ok, so I'm guessing this is probably ok.